snap/info.go

```diff
@@ -311,6 +312,13 @@ type Info struct {
 	// List of system users (usernames) this snap may use. The group of the same
 	// name must also exist.
 	SystemUsernames map[string]*SystemUsernameInfo
+
+	// QuotaGroup is the quota resource group for all apps/services in this
+	// snap. Individual apps may be in different or child groups in addition,
```

Comment:
Based on comment below from @pedronis, this should probably not mention that individual apps can be in subgroups?

Reply:
removed